sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -25,6 +25,24 @@ import { PartitionGate } from "./impl/partitionGate";
 import uuid from "uuid/v4";
 import { validateEventPositions } from "./eventPosition";
 
+/**
+ * Describes the options that can be provided while creating the EventHubConsumerClient.
+ */
+export interface EventHubConsumerClientOptions extends EventHubClientOptions {
+  /**
+   * This setting allows the EventHubConsumerClient to create connections directly to
+   * the Event Hubs partition node.
+   * 
+   * This results in a new connection being created for each partition the client is receiving
```

Comment:
Should we mention why you'd want to enable this? Ie, what are the benefits?

Reply:
I was going to wait for the API review (linked in original comment) before working too hard on the docs here. This PR is really just to show what the API would look like given the proposal (and the benefits are called out in that issue).

This PR should never get merged by the way, but I forgot to set it as draft when I created it :-(